pages/configuration/backup.rst

```diff
@@ -10,3 +10,11 @@ Your Graylog Server setup and settings are easy to backup with a `MongoDB dump <
 
 The data within your Elasticsearch Cluster can take the advantage of the `Snapshot and Restore <https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html>`_ function that are offered by Elasticsearch.
 
+Desaster recovery
+=================
+
+To be able to restore Graylog after a total System crash you need the Graylog ``server.conf`` file - to be exact you need the key that is used in the configuration. The second important part is the MongoDB. This database contains all configuration.
```

Comment:
The key? Which key specifically?

Reply:
We may also want to include the path to server.conf and to where the data is stored in MongoDB. Or else perhaps link to the MongoDB docs that explain how to back up Mongo.